### DISCUSSION TITLE
Automatically copy file nupkg to external folder after Pack (from VS 2022)

### DISCUSSION THREAD

**Discussion by MarcoLLeite** [NuGet/Home#12290] [Q&A]
We have quite a few solutions/projects in separate repositories, so we use a local NuGet source (in our dev machine) for development.

During development we do the changes, build, pack the .nupkg and then copy the file to our Local Nuget folder (from our local NuGet source).

I would like to know if it is possible to copy the .nupkg files after a pack was done automatically, by configuring NuGet or VS 2022 and not the project.
Doing this at the project level would mean too many changes because we have a lot of them.

Any suggestions?

**Comment by gioce90** [comment]
I would LOVE to know if there is some best practice outhere, because I'm in your same situation. I found a solution, I made a Directory.Build.props that can be really useful for you (with some MSBuild directives you can achieve what you ask), but I was asking to myself if there is a better and out-of-the-box way to achieve that.

Moreover, I have a a lot of repositories, each one dedicated to one nuget package... so, copy-paste this Directory.Build.props is a solution, but I would like to find a way to centralize also this file.

**Comment by jeffkl** [comment]
The MSBuild project SDK [Artifacts](https://github.com/microsoft/MSBuildSdks/blob/main/src/Artifacts/README.md) might work for you.  You give it file paths to stage for you after a build and it copies them.

**Comment by gioce90** [comment]
Hi @MarcoLLeite, [NuJet ](https://github.com/gioce90/NuJet) is a project that could help you achieve what you are looking for. Check it out :)
